renderdoccmd/renderdoccmd_android.cpp

```diff
@@ -77,11 +78,15 @@ void handle_cmd(android_app *app, int32_t cmd)
       __android_log_print(ANDROID_LOG_INFO, LOGCAT_TAG,
                           "APP_CMD_INIT_WINDOW: android_state->window: %p", android_state->window);
 
-      const char *argv[] = {
-          "renderdoccmd", "replay", "/sdcard/capture.rdc",
+      char prop[PROP_VALUE_MAX];
+      if(GetAndroidProperty("renderdoc.replay", prop) > 0)
```

Comment:
I don't understand the purpose of this - by definition renderdoccmd is *always* in replay mode/the replay application. We never capture from it, so isn't this check a tautology?